docs/UserGuide.md

```diff
@@ -666,44 +664,47 @@ Examples:
 - `untag Video_1 /lec Lecture_1 /mod CS2040 /tags Yay` removes the tag `Yay` in the video `Video_1` of the
   lecture `Lecture_1` that belongs to the module `CS2040`
 
-## Find
+### Find
 
-### Find Modules or Lectures or Videos
+:exclamation: This is a case insensitive search and matches a target that starts with the search term.
+E.g:
+|Type|Data|Keyword|Matched|
+|-|-|-|-|
+|ModuleCode|[CS2040S, CS2103, ST2334, MA2001]|cs21|[CS2103]|
+|LectureName|[Week 1, Week 2, Week 3]|week|[Week 1, Week 2, Week 3]|
+|VideoName|[Video 1, Video 2, Some video]|video 1, some|[Video 1, Some video]|
 
-> `find {keywords}`
+#### Find Modules
 
-Find all modules/lectures/videos based on context whose code/name (whichever applicable) starts with any of the keyword(s) and case is insensitive.
+> `find {keywords}`
 
-Examples:
+Find all modules whose code starts with any of the keyword(s).
 
-- In root level, `find CS2040S` searches for module `CS2040S` from the module list.
-- In module level within `CS2040S`, `find week 1, week 2` searches for lectures `week 1` or `week 2` from the lecture list of module `CS2040S`.
-- In lecture level within `week2` of `CS2040S`, `find vid1, vid2` searches for videos `vid1` or `vid2` from the video list of lecture `week2` of module `CS2040S`.
+![RootContext](images/RootContext.png)
+has to be appended at the end of the command if you are not in the root context.
 
 ![ModContext](images/ModContext.png)
 ![LectureContext](images/LectureContext.png)
 When in a module or lecture context, the navigation system will inject the `/mod` and `/lec` arguments transforming the user's command into the command specified in [Find Lectures in a Module](#find-lectures-in-a-module) or [Find Videos in a Lecture](#find-videos-in-a-lecture) (refer to [Navigation](#navigation) for more information)
 
-### Find Modules or Lectures or Videos By Tag
+(refer to [Navigation](#navigation) for more information)
+
+#### Find Modules By Tag
 
 > `find {keywords} [/byTag]`
 
-Find all modules/lectures/videos based on context whose tag list contains any tag that starts with any of the keyword(s)
+Find all modules whose tag list contains any tag that starts with any of the keyword(s)
 
 **Assumption:**\
-Module `CS2040S` has tags `["heavy", 'math']`\
-Lecture `Week 1` of `CS2040S` has tags `["Arrays", "Sorting"]`\
-Video `Vid 1` of `Week 1` of `CS2040S` has tags `["content"]`
+Module `CS2040S` has tags `["heavy", 'math']`
```

Comment:
A bit weirdly formatted, with how "Assumption" is bold. Possible to just combine with examples if you feel is necessary to state this?

Reply:
i'll move it into the examples 👍